### DISCUSSION TITLE
posts/Go/gRPC/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#344] [Announcements]
https://www.liwenzhou.com/posts/Go/gRPC/ 

 李文周的Blog gRPC rpc proto 二进制 协议 远程过程调用 thrift Protocol Buffers 微服务

**Comment by nickfan** [comment]
gRPC_demo代码下载url?

**Comment by Q1mi** [comment]
> @nickfan
> gRPC_demo代码下载url?
> 

https://github.com/Q1mi/gRPC_demo

**Comment by jayesslin** [comment]
grpc跟用thrift定义的通信有什么区别吗

**Comment by Q1mi** [comment]
> @jayesslin
> grpc跟用thrift定义的通信有什么区别吗

可以理解为两套轮子

**Comment by Charlotte3240** [comment]
文档可能稍微有一点过期了，生成相应代码的命令可以改成这个
```
protoc --go_out=../services --go_opt=paths=source_relative \
    --go-grpc_out=../services --go-grpc_opt=paths=source_relative \
    Hello.proto
```

**Comment by LMKKK** [comment]
通透!!!
